src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public Ball getRandomBall() {
+        Random randomNumber = new Random();
```

Comment:
1. Not the best name for a variable - this is an instance of `Random`, not a random number itself
2. Please make this a constant or use `Math.random()`